registry/configuring-registry-storage/configuring-registry-storage-openstack-user-infrastructure.adoc

```diff
@@ -0,0 +1,12 @@
+[id="configuring-registry-storage-openstack-user-infrastructure"]
```

Comment:
@adambkaplan since we're only going to have IPI in 4.2, should I comment this content out until it's available?

I did the same with Azure.

Reply:
@bmcelvee that makes sense to me.

Reply:
makes sense

Reply:
Cool, that's done. I've left notes in both assemblies as a reminder that they are commented out and will be updated and allowed to build when each component GAs.